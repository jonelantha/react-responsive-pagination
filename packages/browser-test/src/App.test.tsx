export {};

declare const url: string;

beforeAll(async () => {
  await page.goto(url);

  const totalInput = (await page.$('#totalJson'))!;
  await totalInput.fill('100');
});

// prettier-ignore
const testWidths = [
  1000, 900, 800, 700, 600, 500, 400, 300, 200, 100,
  250, 350, 450, 550, 650, 750, 850, 950,
];

describe.each(testWidths.map(width => [width]))('width (%i)', width => {
  it(`renders correctly at width ${width}px`, async () => {
    await page.setViewportSize({ width, height: 700 });

    await page.evaluate(() => new Promise(requestAnimationFrame));

    const paginationHtml = await page.$eval('ul.pagination', ul => ul.innerHTML);

    expect(paginationHtml).toMatchSnapshot();
  });
});
