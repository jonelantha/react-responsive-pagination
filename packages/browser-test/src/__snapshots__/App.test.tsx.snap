// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`width (100) renders correctly at width 100px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (200) renders correctly at width 200px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (250) renders correctly at width 250px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (300) renders correctly at width 300px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (350) renders correctly at width 350px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (400) renders correctly at width 400px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (450) renders correctly at width 450px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (500) renders correctly at width 500px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (550) renders correctly at width 550px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (600) renders correctly at width 600px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">11</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">12</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (650) renders correctly at width 650px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">11</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">12</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">13</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (700) renders correctly at width 700px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">11</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">12</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">13</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">14</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (750) renders correctly at width 750px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">11</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">12</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">13</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">14</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">15</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (800) renders correctly at width 800px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">11</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">12</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">13</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">14</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">15</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">16</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (850) renders correctly at width 850px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">11</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">12</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">13</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">14</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">15</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">16</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">17</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (900) renders correctly at width 900px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">11</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">12</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">13</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">14</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">15</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">16</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">17</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">18</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">19</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (950) renders correctly at width 950px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">11</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">12</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">13</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">14</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">15</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">16</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">17</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">18</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">19</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">20</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;

exports[`width (1000) renders correctly at width 1000px 1`] = `"<li class=\\"page-item disabled\\"><span class=\\"page-link\\"><span aria-hidden=\\"true\\">«</span><span class=\\"sr-only\\">Previous</span></span></li><li class=\\"page-item active\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"(current)\\"><span aria-hidden=\\"true\\">1</span><span class=\\"sr-only\\">(current)</span></a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">2</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">3</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">4</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">5</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">6</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">7</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">8</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">9</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">10</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">11</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">12</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">13</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">14</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">15</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">16</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">17</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">18</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">19</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">20</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">21</a></li><li class=\\"page-item disabled\\" aria-hidden=\\"true\\"><span class=\\"page-link\\">…</span></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\">100</a></li><li class=\\"page-item\\"><a class=\\"page-link\\" href=\\"#\\" aria-label=\\"Next\\"><span aria-hidden=\\"true\\">»</span><span class=\\"sr-only\\">Next</span></a></li>"`;
